internal/operators/mce/config.go

```diff
@@ -1,8 +1,31 @@
 package mce
 
+import (
+	_ "embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/openshift/assisted-service/internal/common"
+)
+
+type Config struct {
+	MceMinOpenshiftVersion string `envconfig:"MCE_MIN_OPENSHIFT_VERSION" default:"4.10.0"`
+	OcpMceReleaseMatrix    string `envconfig:"OCP_MCE_RELEASE_MATRIX" default:"[{\"openshift_version\": \"4.10\", \"mce_channel\": \"stable-2.3\"}, {\"openshift_version\": \"4.11\", \"mce_channel\": \"stable-2.3\"}, {\"openshift_version\": \"4.12\", \"mce_channel\": \"stable-2.4\"}, {\"openshift_version\": \"4.13\", \"mce_channel\": \"stable-2.4\"}, {\"openshift_version\": \"4.14\", \"mce_channel\": \"stable-2.4\"}, {\"openshift_version\": \"4.15\", \"mce_channel\": \"stable-2.4\"}]"`
+}
+
+type OCPMCEReleaseMatrix struct {
```

Comment:
```suggestion
type OCPMCEVersionMap struct {
```

it doesn't represent a matrix